README.md

```diff
@@ -1,123 +1,25 @@
-# template-microservice
-This serves as a template for the microservices
+# Media Service
 
-## Package structure
+The Media Service is a component designed to fulfill the following functions within the platform:
 
-This package structure is based on multiple sources of best practices in Spring Boot, using roughly the "Package by layer" approach.
-- *root*
-  - *config*
-  - *controller*
-  - *dapr*
-  - *dto*
-  - *exception*
-  - *persistence*
-    - *dao*
-    - *repository*
-    - *mapper*
-  - *service*
-  - *util* (optional, if needed)
-  - *validation*
+1. **Content Upload and Management:** It provides a platform for educators to upload and manage multimedia content, including PDF documents, MP4 videos, PowerPoint presentations (PPTX), and more.
 
-Detailed description of the packages:
+2. **MinIO Object Storage Integration:** To ensure efficient storage and accessibility, the service utilizes the MinIO object storage system. Different types of media are stored in separate buckets within MinIO, allowing for structured storage.
 
-### Root package
+3. **GraphQL Endpoint:** Users can interact with the service through a GraphQL endpoint, enabling the retrieval of URLs for uploading and downloading media files.
 
-This should be named after the microservice itself. This is the root package for the microservice. It contains the `Application.java` file (or of similar name), which is the entry point for the microservice. Usually, this is the only class in this package.
+## Purpose
```

Comment:
Here I would also remove, its almost a repetition of before

Reply:
done @PaulBredl